src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -189,8 +191,42 @@ public void testPermsSeq(TestContext context) {
     });  
    }
 
- @Test
- public void testGroup(TestContext context){
+  @Test
+  public void testPermsUsersPermissionsInvalidUUID(TestContext context)
+      throws InterruptedException, ExecutionException, TimeoutException {
+    String url = "http://localhost:" + port + "/perms/users/123/permissions";
+
+    CompletableFuture<Response> response = new CompletableFuture();
+    send(url, context, HttpMethod.POST, postPermUserPermRequest,
+        SUPPORTED_CONTENT_TYPE_JSON_DEF, 400,  new HTTPResponseHandler(response));
+    Response addPermsResponse = response.get(5, TimeUnit.SECONDS);
+  }
+
+  @Test
+  public void testPermsUsersPutInvalidUUID(TestContext context)
+      throws InterruptedException, ExecutionException, TimeoutException {
+    String url = "http://localhost:" + port + "/perms/users/123";
+
+    CompletableFuture<Response> response = new CompletableFuture();
+    send(url, context, HttpMethod.PUT, postPermUsersRequest,
+        SUPPORTED_CONTENT_TYPE_JSON_DEF, 400,  new HTTPResponseHandler(response));
+    Response addPermsResponse = response.get(5, TimeUnit.SECONDS);
+  }
+
+  @Test
+  public void testPermsUsersDeleteInvalidUUID(TestContext context)
+      throws InterruptedException, ExecutionException, TimeoutException {
+    String url = "http://localhost:" + port + "/perms/users/123";
+
+    CompletableFuture<Response> response = new CompletableFuture();
+    send(url, context, HttpMethod.DELETE, "",
+        SUPPORTED_CONTENT_TYPE_JSON_DEF, 400,  new HTTPResponseHandler(response));
+    Response addPermsResponse = response.get(5, TimeUnit.SECONDS);
```

Comment:
Consider to add this after line 224. Also consider to rename `addPermsResponse` to be DELETE related
`assertEquals(404, addPermsResponse.code);`